packages/core/src/scss/components/checkout/customer/_customer.scss

```diff
@@ -49,9 +49,15 @@
 }
 
 .stripeCustomerEmail-button {
+    @extend .customerEmail-button;
+
     bottom: 3px;
-    padding: $customerEmail-button-padding;
-    width: 100%;
+
+    &--withFloatingLabels {
+        @extend .customerEmail-button;
+
+        bottom: 23px;
```

Comment:
Sorry, forgot to mention, `23px` can be replaced with `spacing("single") + spacing("half")`.
We strongly oppose hardcoding values in SCSS files because it may not be consistent with the existing UI elements.

Reply:
@bc-peng totally understandable, I already tried the change and it works perfectly 🙇‍♂️

Reply:
@bc-peng sorry friend, I think the day I uploaded those changes in my local did not take them correctly (with spacing("single") + spacing("half") the button looks very high, it gives like 2.25 rem), I was playing for a while with the spacing and I found another one, let me know if it's okay for you, I attach an image of how it looks (it gives 1.78125rem)
<img width="1604" alt="image" src="https://user-images.githubusercontent.com/42154828/224160686-35c89a77-b219-46e0-be32-e7dbb9cb6d3a.png">

Reply:
Thanks @AlanMunozN. I just spotted a difference on the screenshots.

![219497079-5d0d2faf-d8ca-4021-973a-64693732e544](https://user-images.githubusercontent.com/88361607/224887987-004a9057-3afa-4515-9abb-1e54aab7d896.png)
![219496950-dfca4dd2-79a3-496e-8f92-169882031982](https://user-images.githubusercontent.com/88361607/224887980-af057ad6-8037-4dcb-a9b2-30ee488b8dae.png)

The space between fields in the checkout has been specified as `8px` by the designer. Can you please match Stripe forms with that? You simply need to use `$floating-form-field-spacing` as the value.

The rest looks good to me.

Reply:
Hey @animesh1987 and @bc-peng, I made changes, and it's ready for another review ♻️